screens/InputScreen.js

```diff
@@ -6,6 +6,10 @@ import InputScanner from '../components/InputScanner';
 import InputManual from '../components/InputManual';
 
 class InputScreen extends React.Component {
+  static navigationOptions = {
+    header: null,
```

Comment:
is this what gets rid of the white space at the top? and how if so?

Reply:
Yes :) It changes the page header using React navigation